CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.8.0
```

Comment:
```suggestion
## Pending
```

Tiny thing but we tend to add unreleased changes to a pending section first, and then whenever a release happens it all gets moved into a heading for the version that it's released in

Reply:
Good to know, I'm also preparing a larger pr to make this package usable for us ;)